frontend/src/scenes/dashboard/DashboardHeader.js

```diff
@@ -69,16 +73,25 @@ export function DashboardHeader({ logic }) {
                                 </Tooltip>
                             ) : null}
 
+                            <Tooltip title={'Share dashboard.'}>
+                                <Button
+                                    className="button-box-when-small enable-dragging-button"
+                                    type={dashboard.is_shared ? 'primary' : ''}
+                                    onClick={() => setShowShareModal(true)}
+                                    data-attr="dashboard-share-button"
+                                >
+                                    <ShareAltOutlined />
+                                    {dashboard.is_shared ? 'Shared' : 'Share dashboard'}
```

Comment:
The "Share Dashboard" label on the button should be hidden in mobile mode:
![image](https://user-images.githubusercontent.com/53387/86569002-1225db00-bf6e-11ea-9048-5b2821071467.png)

Surrounding the text with `<span className="hide-when-small">` should do the trick